helm/strider/ncats-images-meta.yaml

```diff
@@ -4,4 +4,4 @@ strider:
 
 kpRegistry:
```

Comment:
you bumped the version of strider in another PR. please update the version here too.